colossalai/booster/plugin/gemini_plugin.py

```diff
@@ -317,7 +318,8 @@ def __init__(
         max_scale: float = 2**32,
         max_norm: float = 0.0,
         norm_type: float = 2.0,
-        verbose: bool = False,
+        use_tp_pipeline: bool = False,
```

Comment:
How to set tp size? By the way, this naming is a little confusing.

Reply:
Updated. I added the tp_size parameter, and created a MeshGroup to manage the dp_group and tp_group.

Reply:
'use_tp_pipeline' has been modified to 'use_tensor_parallel'

Reply:
updated, CheckpointIO is still under development.